roles/pythondev-site/meta/meta.yml

```diff
@@ -0,0 +1,6 @@
+---
+- dependencies:
+  - { role: common }
+  - { role: python }
+  - { role: nginx }
+  - { role: ssl, domain: {{ pythondev_domain }}, port: {{ pythondev_port }} }
```

Comment:
I am not 100% sure this works as I have it, but it seems like it should - but they may need to be quoted.